crates/subspace-service/src/domains/snap_sync_orchestrator.rs

```diff
@@ -0,0 +1,62 @@
+//! Provides synchronization primitives for consensus and domain chains snap sync.
+
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use subspace_core_primitives::BlockNumber;
+use tokio::sync::broadcast;
+use tokio::sync::broadcast::{Receiver, Sender};
+use tracing::debug;
+
+/// Synchronizes consensus and domain chain snap sync.
+pub struct SnapSyncOrchestrator {
```

Comment:
I thought we agreed that this orchestrator is not needed, yet it still exists. It should be removed as unnecessary.

Reply:
We agreed that we need to provide only two shared variables an atomic bool for the sync oracle and a receiver for the target block. The rest was removed. The current orchestrator is just a container for those variables that are packed together most of the time. It is just more convenient to pass those variables in the same struct even if it is possible to remove it completely.

Reply:
There is also a channel in it. I would have expected to remove this orchestrator completely, have some config data structure in domains if necessary and generally not have methods like `unblock_consensus_snap_sync` that need to be called in strategic moments.